inc/azure_c_logging/xlogging.h

```diff
@@ -82,43 +82,37 @@ typedef void(*LOGGER_LOG_GETLASTERROR)(const char* file, const char* func, int l
+#define LOG_AZ_LOG_WITH_STACK(log_category, log_options, format, ...)                                                                                                   \
 {                                                                                                                                                                       \
     (void)(0 && printf(format, __VA_ARGS__));                                                                                                                           \
     {                                                                                                                                                                   \
-        LOGGER_LOG logger_function = xlogging_get_log_function();                                                                                                                     \
-        if (logger_function != NULL)                                                                                                                                                  \
+        LOGGER_LOG logger_function = xlogging_get_log_function();                                                                                                       \
+        if (logger_function != NULL)                                                                                                                                    \
         {                                                                                                                                                               \
-            char* stackAsString = getStackAsString();                                                                                                                   \
-            if (stackAsString == NULL)                                                                                                                                  \
-            {                                                                                                                                                           \
-                logger_function(log_category, __FILE__, FUNC_NAME, __LINE__, log_options, format, __VA_ARGS__);                                                                       \
+            char stackAsString[STACK_MAX_CHARACTERS];                                                                                                                   \
+            getStackAsString(stackAsString, sizeof(STACK_MAX_CHARACTERS));                                                                                              \
+            size_t formatSize = strlen(format);                                                                                                                         \
+            if(formatSize+sizeof(STACK_PRINT_FORMAT)+1>FORMAT_MAX_CHARACTERS)                                                                                           \
```

Comment:
> if(formatSize+sizeof(STACK_PRINT_FORMAT)+1>FORMAT_MAX_CHARACTERS)    [](start = 11, length = 69)

nit: missing all spaces

Reply:
Added all the spaces

---
In reply to: [453742065](https://github.com/Azure/azure-c-logging/pull/11#discussion_r453742065) [](ancestors = 453742065)